tests/eas/hmp_parity.py

```diff
@@ -400,108 +423,92 @@ def test_first_cpu_migrators(self):
-                    .format(EXPECTED_RESIDENCY_PCT))
+    def test_big_cpus_fully_loaded(self):
+        """Offload Migration and Idle Pull: Big cpus are fully loaded as long as there are tasks left to run in the system"""
+        num_big_cpus = len(self.env.target.bl.bigs)
+
+        end_times = sorted(self.end_times.values())
+
+        # Window of time until the first migrator finishes
+        window = (0, end_times[-num_big_cpus])
+        busy_time = self.a_assert.getCPUBusyTime("cluster",
+                                                 self.env.target.bl.bigs,
+                                                 window=window, percent=True)
+        msg = "Big cpus were not fully loaded while there were enough big tasks to fill them"
+        self.assertGreater(busy_time, EXPECTED_BIG_BUSY_TIME_PCT, msg=msg)
+
+        # As the migrators start finishing, make sure that the tasks
+        # that are left are running on the big cpus
+        for i in range(num_big_cpus-1):
+            big_cpus_left = num_big_cpus - i - 1
+            window = (end_times[-num_big_cpus+i], end_times[-num_big_cpus+i+1])
+            busy_time = self.a_assert.getCPUBusyTime("cluster",
+                                                     self.env.target.bl.bigs,
+                                                     window=window, percent=True)
+
+            expected_busy_time = EXPECTED_BIG_BUSY_TIME_PCT * \
+                                 big_cpus_left / num_big_cpus
+            msg = "Big tasks were not running on big cpus from {} to {}".format(
+                window[0], window[1])
+
+            self.assertGreater(busy_time, expected_busy_time, msg=msg)
+
+    def test_little_cpus_run_tasks(self):
+        """Offload Migration and Idle Pull: Little cpus run tasks while bigs are busy"""
+        tasks = self.params.keys()
+        num_offloaded_tasks = len(tasks) / 2
+
+        first_task_finish_time = None
+        for task in tasks:
+            end_time = self.end_times[task]
+            if not first_task_finish_time or (end_time < first_task_finish_time):
+                first_task_finish_time = end_time
+
+        window = (OFFLOAD_MIGRATION_MIGRATOR_DELAY, first_task_finish_time)
+        busy_time = self.a_assert.getCPUBusyTime("cluster",
+                                                 self.env.target.bl.littles,
+                                                 window=window)
+
+        window_len = window[1] - window[0]
+        expected_busy_time = window_len * num_offloaded_tasks * \
+                             EXPECTED_BIG_BUSY_TIME_PCT / 100.
+        msg = "Little cpus did not pick up big tasks while big cpus were fully loaded"
+
+        self.assertGreater(busy_time, expected_busy_time, msg=msg)
+
+    def test_all_tasks_run_on_a_big_cpu(self):
+        """Offload Migration and Idle Pull: All tasks run on a big cpu at some point
+
+        Note: this test may fail in "golden/silver" systems where the little
```

Comment:
You are right, we shouldn't use the golden/silver terminology.  I've refrased it to:

>         Note: this test may fail in big.LITTLE platforms in which the
>        little cpus are almost as performant as the big ones.